zjit/src/hir.rs

```diff
@@ -446,6 +446,8 @@ pub enum Insn {
     /// Check whether an instance variable exists on `self_val`
     DefinedIvar { self_val: InsnId, id: ID, pushval: VALUE, state: InsnId },
 
+    GetSpecialSymbol { key: u64, svar: u64, state: InsnId },
+    GetSpecialNumber { key: u64, svar: u64, state: InsnId },
```

Comment:
Let's store the `svar` as a new enum -- it seems to not make full use of the u64 and we can have nicer looking code this way